tests/qam-kgraft/update_kgraft.pm

```diff
@@ -81,6 +81,8 @@ sub run() {
     # check if automounter works
     check_automounter;
 
+    script_run("while pgrep packagekitd; do pkcon quit; sleep 1; done");
```

Comment:
or do what we do in most other occasions, e.g. in `tests/console/consoletest_setup.pm`:

```perl
# Stop packagekit
script_run "systemctl mask packagekit.service";
script_run "systemctl stop packagekit.service";
```

or maybe even better to also fix some other potential issues which migh show up later: Just call `consoletest_setup.pm` in before (and `…finish.pm` afterwards).